vendor/fbtransform/transforms/xjs.js

```diff
@@ -11,6 +11,51 @@
 var Syntax = require('jstransform').Syntax;
 var utils = require('jstransform/src/utils');
 
+function commaAfterLastParen(value) {
+    var state = "normal";
+    var commaPos = 0;
+    for (var i=0;i<value.length;++i) {
+        if (state === "normal") {
```

Comment:
Oh then it's probably because it's not being processed - yeah `src/vendor` is in the `eslintignore`.

```
# We can probably lint these later but not important at this point
src/vendor
```

 You could install it globally and run it on a single file or remove the ignore temporarily.